### ISSUE TITLE
Fire Emblem New Mystery of the Emblem Crashes/Graphical bugs

### ISSUE THREAD

**Issue by tom0113** [DS-Homebrew/nds-bootstrap#1158]
**Your console:** New Nintendo 3DS XL
**Launch Method:** SD Card, Luma 10.2.1 - Twilight Menu ++ v20.2.1
**Version used:** Release NDS Bootstrap v0.40.2
**Game tested/used/applicable:** Fire Emblem New Mystery of the Emblem Original Japanese Version, Title ID: VI2J, MD5: 7C5BE815DEF8FB6C00231CB82F492C8A

**Expected Behavior:**  No graphical glitches, after a character's line of dialogue finishes you press L to open the guide
**Actual Behavior:** Upon loading prologue 1, graphical glitches occur on the touch screen, and sometimes on the top screen, if L is pressed to open the guide the game crashes

### Steps to reproduce
Start a new file and proceed to prologue 1, graphical glitches occur, after a character's line of dialogue finishes you press L to open the guide and the game crashes

### Logs
[NDSBTSRP.LOG](https://github.com/DS-Homebrew/nds-bootstrap/files/6527767/NDSBTSRP.LOG)

### Other notes
Same bug occured/occurs on English translation patch, I tested on original to see if it was an issue with the patch and the bug still occurs
Bug does not occur on NDS Bootsrap v0.39.0

**Comment by tom0113** [comment]
Since these issues didn't happen previously I've tested and none of these issues occur in NDS Boostrap v0.39.0

**Comment by RocketRobz** [comment]
Should already be fixed in the latest nightly.

**Comment by tom0113** [comment]
Unsure if I should start a new issue for this so I kept it in the original thread due to the issues being related.
While the issue is fixed in the nightly version, the logo on the top screen of the game's title screen has graphical bugs now, which did not occur in the Release NDS Bootstrap v0.40.2 version. That's the only issue I found on the original Japanese version of the game, and the issues I mentioned in the original post no longer occur.
On a translation patched english version (this one: https://forums.serenesforest.net/index.php?/topic/69176-updated-fe12-translation-patch-beta-2-released/), the title screen graphic displays properly unlike the Japanese version, however pressing start/a crashes the game.
Version used was v0.40.2-2485ff2
I was also unsure on whether the latest version which the issue guidelines said to use was referring to the nightly or release version, I'd assume based on your reply it refers to the release version?
Thanks for your help, if there's anything that needs clarifying let me know
